mobile/lib/features/trade/domain/trade_values.dart

```diff
@@ -163,6 +163,6 @@ class TradeValues {
   }
 
   _recalculateMaxQuantity() {
-    maxQuantity = tradeValuesService.calculateMaxQuantity(price: price, leverage: leverage);
```

Comment:
Should be fixed with https://github.com/get10101/10101/pull/2362

Reply:
I will drop the commit before merging!